eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/package-info.java

```diff
@@ -25,6 +25,9 @@
  *     <li>
  *         {@link org.eclipse.collections.api.list.FixedSizeList} - a list that may be mutated, but cannot grow or shrink in size.
  *     </li>
+ *     <li>
+ *         {@link org.eclipse.collections.api.list.ParallelListIterable} - a {@link org.eclipse.collections.api.RichIterable} which will defer evaluation for certain methods like select, reject, collect, etc.
```

Comment:
Please add this as well `Evaluation occurs in parallel`. So the final description will look like this:
`{@link org.eclipse.collections.api.list.ParallelListIterable} - a {@link org.eclipse.collections.api.RichIterable} which will defer evaluation for certain methods like select, reject, collect, etc. When evaluation occurs it is in parallel.`

Reply:
Hi @nikhilnanivadekar  please review .

Reply:
LGTM